From 8b8ad6245ec0c3b1e8bee270d648c99108199633 Mon Sep 17 00:00:00 2001
From: Markus Partheymueller <markus.partheymueller@cyberus-technology.de>
Date: Mon, 23 Sep 2019 13:57:21 +0200
Subject: [PATCH 1/4] multiboot: Increase module count and cmdline length.

---
 src/arch/x86/image/multiboot.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/arch/x86/image/multiboot.c b/src/arch/x86/image/multiboot.c
index 0c85df70..146eb5ca 100644
--- a/src/arch/x86/image/multiboot.c
+++ b/src/arch/x86/image/multiboot.c
@@ -57,7 +57,7 @@ FEATURE ( FEATURE_IMAGE, "MBOOT", DHCP_EB_FEATURE_MULTIBOOT, 1 );
  * physical addressing as per the multiboot specification.
  *
  */
-#define MAX_MODULES 8
+#define MAX_MODULES 16
 
 /**
  * Maximum combined length of command lines
@@ -67,7 +67,7 @@ FEATURE ( FEATURE_IMAGE, "MBOOT", DHCP_EB_FEATURE_MULTIBOOT, 1 );
  * virt_to_phys(cmdline) to point to the command lines, even though
  * this would comply with the Multiboot spec.
  */
-#define MB_MAX_CMDLINE 512
+#define MB_MAX_CMDLINE 4096
 
 /** Multiboot flags that we support */
 #define MB_SUPPORTED_FLAGS ( MB_FLAG_PGALIGN | MB_FLAG_MEMMAP | \
-- 
2.25.1

